applications: asset_tracker_v2: update config handling for nRF cloud devices (#6900)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

nRF9160: Asset Tracker v2
-------------------------

* Updated:

>   * For nRF Cloud builds, the configuration section in the shadow will now be initialized during the cloud connection process.

Reviewer: ```suggestion
  * For nRF Cloud builds, the configuration section in the shadow is now initialized during the cloud connection process.
```

This section provides detailed lists of changes by :ref:`library <libraries>`.

Libraries for networking
------------------------

>   * :ref:`lib_nrf_cloud` library:
> 
>     * Updated:
> 
>       * During the connection process, shadow data is sent to the application even if no "config" section is present.
>       * The application can now send shadow updates earlier in the connection process.

Reviewer: ```suggestion
* :ref:`lib_nrf_cloud` library:

  * Updated:

     * During the connection process, shadow data is sent to the application even if no "config" section is present.
     * The application can now send shadow updates earlier in the connection process.
```
Just minor alignments
